Add HTML and PNG versions of createRole flowchart

afig opened the pull request:
This flowchart describes the sequence in which the `createRole()` function located in `addRoleBaseMgmtCore.sql` performs its checks and what corresponding actions are taken.

For more information, see issue #136

smurthys commented:
Thanks for adding the files @afig.

I think the following improvements are possible:
- Change HTML title to something like `ClassDB Role-Creation Process`
- Add a heading to the chart with the same text as the document title. This will be similar to what is done in the ERD.
- I like the idea of the current text showing the signature of createRole. Perhaps that could be shown as the sub-heading in the same font size as now but saying `Function createRole(roleName,...initialPwd)`
- One thing in ERD that is very helpful is to show somewhere that the chart is drawn using draw.io. I don't quite know where that fits.

afig commented:
Thanks for the recommendations @smurthys. I took them into account and updated the title, and added one within the diagram. I also tweaked the text that showed the signature of `createRole`.

I split the last line of the attribution box in order to reduce the box's overall width. I was then able to add a draw.io mention at the bottom of the page without it sticking out too much.

smurthys commented:
I think these changes will do.
